GitLab-Groups-Group/.rpdk-config

```diff
@@ -24,5 +24,16 @@
         "codegen_template_path": "default",
         "protocolVersion": "2.0.0"
     },
+    "contractSettings":{
+        "typeConfiguration": {
+            "configuration": {
+                "GitLabAccess": {
+                    "AccessToken": "{{resolve:ssm-secure:/cfn/gitlab/token:5}}",
```

Comment:
Same comment as before on the secret version. Instead of commenting on any others if you make this change I would make it to all the `.rpdk-config`